eventdisplay/tracks/FairEveGeoTracks.cxx

```diff
@@ -49,8 +49,10 @@ InitStatus FairEveGeoTracks::Init()
 {
     FairRootManager *mngr = FairRootManager::Instance();
     fContainer = (TClonesArray *)mngr->GetObject("GeoTracks");
-    if (fContainer == nullptr)
-        return kFATAL;
+    if (fContainer == nullptr) {
```

Comment:
```suggestion
    if (!fContainer) {
```

Let's use this opportunity and apply [ES.87](https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#es87-dont-add-redundant--or--to-conditions). Thx!